jbpm-test/src/main/java/org/jbpm/test/JbpmJUnitBaseTestCase.java

```diff
@@ -933,6 +936,10 @@ public void addEnvironmentEntry(String name, Object value) {
     	customEnvironmentEntries.put(name, value);
     }
 
+    public void addPersistenceProperty(String name, Object value) {
```

Comment:
Better name for this would be ```setPersistenceProperty``` - this way, it is made clear that using the same ```name``` twice will override the previous value.